liste-envies-front/src/app/component/wish-card/wish-card.component.html

```diff
@@ -324,7 +324,7 @@ <h6 matLine><span [innerHTML]="comment?.text"></span></h6>
 
     <mat-label
       color="warn"
-      *ngIf="wishItem.userTake"
+      *ngIf="wishItem.userTake && wishItem.userTake.length > 0"
```

Comment:
Cela peut être juste 

```suggestion
      *ngIf="wishItem.userTake?.length > 0"
```